src/lib/metaphysics.ts

```diff
@@ -0,0 +1,60 @@
+interface Args {
+  /** GraphQL query */
+  query: string
+
+  /** GraphQL variables */
+  variables: Record<string, unknown>
+
+  /**
+   * HTTP Headers
+   *
+   * If additional headers such as `X-Access-Token` or `X-XApp-Token` are required,
+   * provide them here.
+   *
+   */
+  headers: Record<string, string>
+}
+
+/**
+ * Helper to make a request to Metaphysics
+ */
+
+export async function metaphysics(args: Args) {
+  const { query, variables } = args
```

Comment:
We can re-use this now, instead of reinventing it each time. A good time to add better types and error handling.

Reply:
Will definitely need your help connecting the dots if/when we get to that point. Not clear to me how (or even why?) we would use Relay for this kind of server-side imperative code.

Reply:
Generated types (!), caching, familiar API, relay network layer for debugging, and so much more.